docs/blender_docs/io_gltf2.rst

```diff
@@ -30,7 +30,198 @@ This importer/exporter supports the following glTF 2.0 features:
+in Blender, as Blender has other ways of showing ambient occlusion, but this
+method will allow the exporter to write an occlusion image to the glTF.
+
+glTF stores occlusion in the Red (``R``) channel, allowing it to optionally share
+the same image with the Roughness and Metallic channels.
+
+Normal Map
+^^^^^^^^^^
+
+To use a Normal Map in glTF, connect an "Image Texture" node's color output
+to a "Normal Map" node's color input, and then connect the "Normal Map" normal
+output to the "Principled BSDF" node's "Normal" input.  The Image Texture node
+for this should have its "Colorspace" setting configured to "Non-Color Data".
+
+The "Normal Map" node must remain on its default setting of "Tangent Space" as
+this is the only type of normal map currently supported by glTF.  The strength
+of the normal map can be adjusted on this node.  The exporter isn't exporting
+these nodes directly, but will use them to locate the correct image and will
+copy the strength setting into the glTF.
+
+.. figure:: /images/addons_io-gltf2-material-normal.png
+   :alt: A normal map image connected such that the exporter will find it and copy it
+         to the glTF file.
+
+   A normal map image connected such that the exporter will find it and copy it
+   to the glTF file.
+
+.. note::
+
+   Blender's "Cycles" rendering engine has a "Bake" panel that can be used to bake
+   tangent-space normal maps from almost any other arrangement of normal vector
+   nodes.  Switch the "Bake type" to "Normal".  Keep the default space settings
+   (Space: Tangent, R: +X, G: +Y, B: +Z) when using this bake panel for glTF.
+   The resulting baked image can be saved and hooked up to a new material using
+   the Normal Map node as described above.
+
+Emissive
+^^^^^^^^
+
+An "Image Texture" node can be connected to an "Emission" shader node, and optionally
+combined with settings from a "Principled BSDF" node by way of an "Add" shader node.
+
+If the glTF exporter finds an image connected to the Emission shader node, it will
+export that image as the glTF material's emissive image.
+
+Double Sided
+^^^^^^^^^^^^
+
+In glTF, double-sided is a property that is applied per-material, not per-viewport
+or per-mesh, so it has no exact equivalent within Blender.  It can be thought of as
+a combination of backface culling (in Blender's viewport) and double-sided lighting
+(a Blender mesh property).
+
+When ``false`` (the default), backface culling is used, and the backs of faces in
+the glTF will not be visible in other software.  When ``true``, backface culling
+is disabled, and double-sided lighting is used, automatically reversing the normal
+vectors of any visible back faces.
+
+To set this value to true, create a custom node group by the name of
+``glTF Metallic Roughness``, add an input value named ``DoubleSided`` with a range
+of 0.0 to 1.0, and set it to 1.0.  There will be no equivalent effect in Blender,
+but the exporter will enable double-sided mode in glTF for this material.
```

Comment:
Per above, I'm not sure how to create a custom node group. Long term I still consider this a bug to be fixed in the exporter (by duplicating materials and applying mesh double-sided settings to them, if necessary) rather than something we should guide users through hoops to work around. It seems unlikely that the Principled BSDF node will ever have a doubleSided input.

Reply:
When the bug gets fixed, I'll update this section too.